README.md

```diff
@@ -81,12 +79,58 @@ gem build logstash-filter-awesome.gemspec
 # Logstash 2.3 and higher
 bin/logstash-plugin install --no-verify
 
-# Prior to Logstash 2.3
-bin/plugin install --no-verify
 
 ```
 - Start Logstash and proceed to test the plugin
 
+## Added features:
```

Comment:
Remove this section. In the PR to logstash HTTP input, we will explain what we added.